File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, s: str) -> bool:
        d={}
        for i in range(len(s)):
            if s[i] not in d:
                d[s[i]]=1 
            else:
                d[s[i]]+=1 
        if len(d)==26:
            return True 
        return False
        

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string s) {
        unordered_map<char,int>mp;
        for(int i=0;i<s.size();i++)
            mp[s[i]]++;
        string a="abcdefghijklmnopqrstuvwxyz";
        for(int i=0;i<26;i++)
        {
            if(mp.find(a[i])!=mp.end())
                continue;
            else
                return false;
        }
        return true;
    }
};